webui/main.py

```diff
@@ -49,3 +58,78 @@ def add_completion_task(req_body: RequestBody):
 def get_completion_tasks(page: int = 0):
     completion_tasks = db_get_completion_tasks_by_page(page)
     return {"tasks": completion_tasks}
+
+
+active_connections = {}
+
+
+@app.websocket("/ws")
+async def on_connection(websocket: WebSocket):
+    await websocket.accept()
+    active_connections[websocket] = []
+    print(active_connections)
```

Comment:
It's ok to have print statements, but please give them more verbosity instead of printing only an object with no context, or remove it all together if you think it's not necessary anymore.